tests/test_nim_benchmark_release_acceptance.py

```
@@ -83,13 +83,19 @@ def test_package_import_does_not_eagerly_load_optional_benchmark() -> None:
+    tmp_path: Path, monkeypatch: pytest.MonkeyPatch
+) -> None:
     """Schema-demo prices can support dry runs but can never drive a live policy."""
+    monkeypatch.setattr(nb, "_require_current_actual_cost_evidence", lambda: None)
     register_credential(nb.NIM_CREDENTIAL_NAME, "secret-test-key")
     scenario = json.loads(EXAMPLE_PRICING_PATH.read_text(encoding="utf-8"))
     scenario_path = _write_json(tmp_path / "unreviewed_pricing.json", scenario)
 
-    with pytest.raises(nb.BenchmarkContractError, match="reviewed"):
+    with pytest.raises(
+        nb.BenchmarkContractError,
+        match="^live benchmark pricing scenario must be independently reviewed$",
```

Ruff (0.16.3)

[warning] Pattern passed to `match=` contains metacharacters but is neither escaped nor raw

(RUF043)